packages/global-search-module/src/index.ts

```diff
@@ -0,0 +1,50 @@
+import { defineModule } from '@directus/extensions-sdk';
+import ModuleComponent from './module.vue';
+import Settings from './settings.vue';
+
+declare global {
+	interface Window {
+		globalSearchListenerAdded: boolean;
+	}
+}
+
+function handleCmdK(event: KeyboardEvent, router: { name: string; }[]) {
+	if (event.metaKey && event.key === 'k') {
```

Comment:
Sadly `metaKey` is not `Ctrl` on windows, but rather the `Win` key.

So this need to do some OS detection :/

Reply:
boooo 🤣

Reply:
Will need to fix that here as well. I should have looked at how we handle this in core.

https://github.com/directus-labs/extensions/blob/ba7ee7823094dcdce6a5b39d6162922b76a020b1/packages/global-search-module/src/settings.vue#L78-L85

Reply:
Hmm in core we just don't care if you pressed either Ctrl or Meta. So that's that. Probably not correct either, but at least we don't have to do OS detection then